src/colaa.cpp

```diff
@@ -175,6 +177,22 @@ void CoLaA::setEchoFilter(CoLaAEchoFilter::EchoFilter filter)
   readBack();
 }
 
+void CoLaA::setParticleFilter(bool particle_filter)
+{
+  std::stringstream cmd;
+  cmd << SET_PARTICLE_FILTER_COMMAND << " " << std::to_string(static_cast<bool>(particle_filter)) << " " << std::to_string(static_cast<int>(500));
```

Comment:
is the `500` something that is given by the protocol or should it be a parameter?

Reply:
it is a fixed number in protocol. don't know why sick has even implemented this. maybe it is a variable in a later release for colaa.